### PR TITLE
Allow creation of Resources through endpoint

### PR DESCRIPTION
**Pull request by dsgoers**

### CODE REVIEW

File: app/controllers/resources_controller.rb [hunk 1]
```diff
@@ -39,6 +44,6 @@ def load_resource
   end
 
   def permitted_params
-    [:name, :abbreviation, :manifest, :onesky_project_id, :system_id, :description]
+    [:name, :abbreviation, :manifest, :onesky_project_id, :system_id, :description, :resource_type_id]
```
Comment (knutsenm) [inline app/controllers/resources_controller.rb]: Does it matter that you are now allowed to change resource_type_id on update?

Comment (dsgoers) [inline app/controllers/resources_controller.rb]: Yea I was thinking about that and I don't see any issues right now.

**Comment by dsgoers** [comment]
@knutsenm please review the new commit 😄

File: config/routes.rb [hunk 2]
```diff
@@ -3,7 +3,7 @@
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
 
   resources :systems, :auth, :translations, :drafts, :resources, :custom_pages, :languages, :attributes,
-            :translated_attributes, :pages, :views, :attachments, :translated_pages, :follow_ups
+            :translated_attributes, :pages, :views, :attachments, :translated_pages, :follow_ups, :resource_types
```
Comment (knutsenm) [inline config/routes.rb]: Should this one be `, only: [:index]` ?

If so, are there any other routes you should be restricting?

**Comment by dsgoers** [comment]
thanks!!

File: config/routes.rb (hunk 2 above)
Comment (dsgoers) [inline config/routes.rb]: I think it's OK like that.  404 is the response if an unimplemented method is called

**Pull request merged**